man/ostree-config.xml

```diff
@@ -50,9 +50,15 @@ Boston, MA 02111-1307, USA.
     </refnamediv>
 
     <refsynopsisdiv>
+            <cmdsynopsis>
+                <command>ostree config get</command> <arg choice="req">SECTIONNAME.KEYNAME</arg>
```

Comment:
Do we want to encourage users to us dots in future? I thought we will keep existing Dot usecase to maintain backward compatibility and we suggest future usage with using --group option.

Reply:
Hmm yeah, I think it should be fine to support/encourage it. It's really just the `remote "foobar"` case that makes things annoying. In other situations, I'd much rather type `ostree config get core.config` instead of `ostree config get --group=core config`.